Data/Repositories/ExerciseRepository.cs

```diff
@@ -0,0 +1,83 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+
+using FitnessTracker.Data.Entities;
+
+namespace FitnessTracker.Data.Repositories
+{
+    public class ExerciseRepository : IExerciseRepository
+    {
+        private readonly FitnessTrackerContext _db;
+
+        public ExerciseRepository(FitnessTrackerContext db)
+        {
+            _db = db;
+        }
+
+        public Exercise Create(Exercise exercise)
+        {
+            _db.Attach(exercise).State = EntityState.Added;
+            _db.SaveChanges();
+            return exercise;
+        }
+
+        public Exercise Delete(int id)
+        {
+            if (GetExerciseById(id) == null) return null;
+            
+            var resdults = _db.Remove(new Exercise { ExerciseId = id }).Entity;
```

Comment:
Why not just hold onto the exercise from GetExerciseById instead of create a new entity to delete?